packages/utils/upgrade/src/__tests__/core/format.test.ts

```diff
@@ -0,0 +1,85 @@
+import * as f from '../../core/format';
+
+import type { RunReports } from '../../types';
+
+describe('Format', () => {
+  test('Path', () => {
+    const formatted = f.path('Foo');
+    expect(formatted).toStrictEqual(formatted);
```

Comment:
Just noticed this doesn't check anything :face_in_clouds:
I'll update it tomorrow

Reply:
Done